### PR TITLE
0.8 port

### PR DESCRIPTION
**Pull request by atotic**

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -24,10 +24,11 @@
     "google"
   ],
   "dependencies": {
-    "polymer": "Polymer/polymer#>=0.5.2 <1.0.0",
-    "google-apis": "GoogleWebComponents/google-apis#~0.4.3"
+    "google-apis": "GoogleWebComponents/google-apis#0.8-preview",
+    "polymer": "Polymer/polymer#0-8.preview"
```
Comment (ebidel) [inline bower.json]: `0.8-preview`

Comment (atotic) [inline bower.json]: fixed

File: demo.html [hunk 2]
```diff
@@ -5,73 +5,40 @@
...
+        <strong>Short URL</strong>:
+        <a href="{{shortUrl}}" target="_blank">{{shortUrl}}</a>
+      </p>
+      <p>
+        <p><strong>Error: </strong><span style="color:red">{{urlError}}</span></p>
+      </p>
+    </form>
+  </template>
+  <script>
+    document.querySelector('template').shorten = function(ev) {
```
Comment (ebidel) [inline demo.html]: ev -> 3

File: google-url-shortener.html [hunk 3]
```diff
@@ -83,8 +87,32 @@
...
+          value: '',
+          notify: true
+        },
 
+        /** 
+         * Error when url was shortened
+         * @attribute error
+         * @default ''
+         */
+        error:{
```
Comment (ebidel) [inline google-url-shortener.html]: spacing `error: {`

File: google-url-shortener.html [hunk 4]
```diff
@@ -93,59 +121,58 @@
...
+      /**
+       * Shortens the URL in `longUrl`.
+       *
+       * @method shorten
+        */
+      shorten: function() {
+        if (apiLoaded_) {
+          if (this.longUrl) {
+            var request = this.$.urlshortener.api.url.insert(
+                { 'resource': { 'longUrl': this.longUrl }});
```
Comment (ebidel) [inline google-url-shortener.html]: spacing : `{resource: {longUrl: this.longUrl}}`

File: google-url-shortener.html [hunk 5]
```diff
@@ -93,59 +121,58 @@
...
+
+            request.execute(function(response) {
+              if (response && response.id) {
+                this.shortUrl = response.id;
+                this.error = '';
+                this.fire('google-url-shorten', { shortUrl: response.id });
```
Comment (ebidel) [inline google-url-shortener.html]: no spacing padding. When there's doubt, refer to https://google-styleguide.googlecode.com/svn/trunk/javascriptguide.xml

**Comment by ebidel** [comment]
Reviewed

File: demo.html (hunk 2 above)
Comment (atotic) [inline demo.html]: hate e. done.

File: google-url-shortener.html (hunk 3 above)
Comment (atotic) [inline google-url-shortener.html]: done.

File: google-url-shortener.html (hunk 4 above)
Comment (atotic) [inline google-url-shortener.html]: done

**Comment by ebidel** [comment]
LGTM

**Pull request merged**